### ISSUE TITLE
Move from SQLite to something else, like MySQL

### ISSUE THREAD

**Issue by cor** [woording/woording-api#17]
SQLite won't scale at all, and will be very bad at handeling multiple users at the same time.  
We need to start using something else, like MySQL.

**Comment by mikevanb** [comment]
Will need to decide and implement which database to use as this will determine what methods are used to prevent injection vulnerabilities.

**Comment by cor** [comment]
Good point, we can however implement input validation (i. e. check if there are semicolons in the username and such) already, because this won't be affected by the database type we choose since the SQL syntax is 99% the same.

**Comment by cor** [comment]
We'll move this to a dedicated docker container, see `woording/woording-db`
